specification/loadtestservice/resource-manager/readme.md

```diff
@@ -74,7 +98,6 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-python
```

Comment:
Since python already supports SDK generation from typespec, there is no need to keep python SDK configuration for swagger.